Fix pinned note separation

martinstacey opened the pull request:
### Purpose
This PR makes sure the distance from pinned node to note respects the new UI warning size.
It does it by changing `DISTANCE_TO_PINNED_NODE_WITH_WARNING` from 36 to 64.

Before PR:
![image](https://user-images.githubusercontent.com/25138291/139928371-3f3819d7-adde-47ea-bc91-690113de7550.png)
After PR:
![image](https://user-images.githubusercontent.com/25138291/139928394-fd3f7180-cd26-4821-9860-e96fec3d06bc.png)

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

@QilongTang @Amoursol 
### FYIs

@SHKnudsen @OliverEGreen

QilongTang commented:
@martinstacey Regression to look at: `DynamoCoreWpfTests.NoteViewTests.NoteIsPlacedAbovePinnedNodeInErrorState`

martinstacey commented:
> @martinstacey Regression to look at: `DynamoCoreWpfTests.NoteViewTests.NoteIsPlacedAbovePinnedNodeInErrorState`

ok @QilongTang, fixed it on commit d24e8c4, I updated the test to reflect the existing distance from note to node
